src/main/java/com/vinhderful/raytracer/Benchmark.java

```diff
@@ -138,47 +138,70 @@ public static void main(String[] args) throws Exception {
         // Choose device
         System.out.println("-----------------------------------------");
 
-        for (int deviceId=0; deviceId<devices.size(); deviceId++) {
-            TornadoDevice device = devices.get(deviceId);
+        // ==============================================================
+        // Run computation sequentially
+        // ==============================================================
+        System.out.println("-----------------------------------------");
+        System.out.println("Running [JAVA SEQUENTIAL]");
+        for (int i = 0; i < FRAMES_TO_GENERATE; i++)
+            Renderer.render(pixels, dimensions, camera, rayTracingProperties,
+                    bodyPositions, bodySizes, bodyColors, bodyReflectivities,
+                    skybox, skyboxDimensions);
+
+        long startTime = System.nanoTime();
+        Renderer.render(pixels, dimensions, camera, rayTracingProperties,
+                bodyPositions, bodySizes, bodyColors, bodyReflectivities,
+                skybox, skyboxDimensions);
+        long endTime = System.nanoTime();
+        double sequentialTime = (endTime - startTime) / 1000000.0;
+        System.out.println("Duration: " + sequentialTime + " ms");
+
+
+        // ==============================================================
+        // Run with Java Parallel Streams
+        // ==============================================================
+        System.out.println("-----------------------------------------");
+        System.out.println("Running [JAVA PARALLEL STREAMS]");
 
+        for (int i = 0; i < FRAMES_TO_GENERATE; i++)
+            Renderer.renderWithParallelStreams(pixels, dimensions, camera, rayTracingProperties,
+                    bodyPositions, bodySizes, bodyColors, bodyReflectivities,
+                    skybox, skyboxDimensions);
+
+        startTime = System.nanoTime();
+        Renderer.renderWithParallelStreams(pixels, dimensions, camera, rayTracingProperties,
+                bodyPositions, bodySizes, bodyColors, bodyReflectivities,
+                skybox, skyboxDimensions);
+        endTime = System.nanoTime();
+        double javaStreamsTime = (endTime - startTime) / 1000000.0;
+        System.out.println("Duration: " + javaStreamsTime + " ms");
```

Comment:
This will be executed also in the Jenkins pipeline. Do we want JAVA PARALLEL STREAMS in Jenkins?

Otherwise, I suggest to create a different class, or rename the current one as `RegressionTests.java` and execute as:
```tornadovm-ray-tracer regression```

Reply:
Yes, it executes the parallel streams as a reference

Reply:
we do not need this in our regression tests, right?

Reply:
I think there is no need for separation. The PR removes redundant executions

Reply:
ok, after the sync, we provide the following:

```bash
./bin/tornadovm-ray-tracer regression
```

We can use this one in our Jenkins server. It will skip the sequential code and compare it against the Java parallel streams version.